common-lib/src/main/scala/com/gu/mediaservice/lib/auth/Store.scala

```diff
@@ -66,6 +66,7 @@ class KeyStore(bucket: String, credentials: AWSCredentials) extends BaseStore[St
 object PermissionType extends Enumeration {
   type PermissionType = Value
   val EditMetadata = Value("editMetadata")
+  val DeleteImage  = Value("deleteImage")
```

Comment:
Currently nobody has this permission, which is OK. Need to check if the list would initially be the same as `editMetadata`.